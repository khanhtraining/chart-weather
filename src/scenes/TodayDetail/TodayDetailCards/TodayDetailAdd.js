import React from 'react'
import './todaydetailcards.scss'

export const TodayDetailAdd = () => {
    return (
        <div className='todayDetailCards-wrapper-item'>
            <div className='add-icon'></div>
            <div className='add-text'>Add</div>
        </div>
    )
}

export default TodayDetailAdd
