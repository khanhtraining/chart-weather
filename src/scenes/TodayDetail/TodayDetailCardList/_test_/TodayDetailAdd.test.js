import React from 'react'
import renderer from 'react-test-renderer'
import TodayDetailAdd from '../TodayDetailAdd'

describe('snapshot testing', () => {
    test('snapshot for today detail add component', () => {
        const renderedComponent = renderer.create(<TodayDetailAdd/>).toJSON()
        expect(renderedComponent).toMatchSnapshot()
    })
})
