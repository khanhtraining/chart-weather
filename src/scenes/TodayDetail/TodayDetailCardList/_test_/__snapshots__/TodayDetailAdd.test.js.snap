// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing snapshot for Today Detail Add component 1`] = `
<div
  className="todayDetailCards-wrapper-item"
>
  <div
    className="add-icon"
  />
  <div
    className="add-text"
  >
    Add
  </div>
</div>
`;

exports[`snapshot testing snapshot for Today Detail component 1`] = `
<div
  className="todayDetailCards-wrapper-item"
>
  <div
    className="add-icon"
  />
  <div
    className="add-text"
  >
    Add
  </div>
</div>
`;

exports[`snapshot testing snapshot for today detail add component 1`] = `
<div
  className="todayDetailCards-wrapper-item"
>
  <div
    className="add-icon"
  />
  <div
    className="add-text"
  >
    Add
  </div>
</div>
`;
